iterator/linked-list.py

```diff
@@ -0,0 +1,136 @@
+class Node:
```

Comment:
VERIFICATION RESULT

Your pull request: ITERATOR-1021 Added linked-list.py


Your commit: Create linked-list.py
The prefix of your comment must be from ['LEETCODE', 'GENERATOR', 'TRIANGLE', 'HEXNUMBER', 'ITERATOR', 'REQUESTS']
Number of your group must be from ['1021', '1022']